packages/react-query/src/useQueries.ts

```diff
@@ -188,5 +190,35 @@ export function useQueries<T extends any[]>({
     observer.setQueries(defaultedQueries, { listeners: false })
   }, [defaultedQueries, observer])
 
+  const errorResetBoundary = useQueryErrorResetBoundary()
+
+  defaultedQueries.forEach((query) => {
```

Comment:
can we rename this from `queries` to `options` ? I think this is highly confusing right now 😅

Reply:
[237912b](https://github.com/TanStack/query/pull/4177/commits/237912b2459af5df2397416a57cb185ac3dc534f)

Reply:
sorry, I meant more globally that `defaultedQueries` should also probably be `defaultedOptions`. It's not a query 😅

Reply:
okay nevermind, this is actually a bigger refactoring. we have `queries` everywhere and also `setQueries`...

Reply:
I was struggling to grasp the naming conventions here, and when I thought I finally got it, this comment got me confused again 😁

Let me explain how I understand it. First, here are 3 facts about the code in this PR its current state:
1. `useQueries` takes `queries` array as an argument
```ts
export function useQueries<T extends any[]>({
  queries: queriesOptions,
  context,
}: {
  queries: readonly [...QueriesOptions<T>]
  context?: UseQueryOptions['context']
})
```
2. Each item of `queries` array is a _query_, but _query_ is basically an `options` object
3. These `queries` are then _defaulted_ and stored as `defaultedQueries`:
```ts
const defaultedQueries = React.useMemo(
  () =>
    queriesOptions.map((options) => {
      const defaultedOptions = queryClient.defaultQueryOptions(options)

      // Make sure the results are already in fetching state before subscribing or updating options
      defaultedOptions._optimisticResults = isRestoring
        ? 'isRestoring'
        : 'optimistic'

      return defaultedOptions
    }),
  [queriesOptions, queryClient, isRestoring],
)
```

It would be strange if mapping with default values would suddenly convert `queries` to `options`. Furthermore in the code below `QueriesObserver` constructor has second parameter named `queries` and not `options` (although `queries` is typed as `QueryObserverOptions`, because why not xD).
```ts
const [observer] = React.useState(
  () => new QueriesObserver(queryClient, defaultedQueries),
)
```


So here are my thoughts to sum it up:
If we rename `defaultedQueries` to `defaultedOptions` it would be confusing because `options` are of a single query and not of `queries` array (`useQueries` does not have `options` on its own). To make it more consistent and less confusing, I would instead suggest to take the example from how `queries` are typed in `useQueries` arguments (type is called `QueriesOptions`) and do the name change this way:

`defaultedQueries` -> `defaultedQueriesOptions`

WDYT?

If that makes sense, maybe we could also do this to make it even more consistent:
```ts
export function useQueries<T extends any[]>({
  queries: queriesOptions,
  context,
}
```

Reply:
yeah the naming is weird, I agree. But we have a public method called `setQueries` that actually takes `options` in, but they are named `queries` :/

so for now, let's:

- revert the lats commit (sorry) and just keep it `queries` for now. Being consistent is better I think.

there are also conflicts now, sorry. If those are solved we can go ahead and merge 🚀